ZipBuilder/Sources/ZipBuilder/Subspec.swift

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/// All the subspecs available in the Firebase pod.
+public enum Subspec: String {
```

Comment:
An enhancement would be to generate this list from the Firebase pod.  Otherwise, this will be one more place to keep in sync when we add and remove Firebase components.

Reply:
I like that idea. Adding a `TODO`.